runtime/compiler/control/CompilationRuntime.hpp

```diff
@@ -628,6 +628,18 @@ class CompilationInfo
       method->extra = reinterpret_cast<void *>(value);
       }
 
+#if defined(J9VM_OPT_MICROJIT)
+   static void setInitialMJITCountUnsynchronized(J9Method *method, int32_t mjitThreshold, int32_t trThreshold)
+      {
+      int32_t value = 0;
+      if(mjitThreshold < trThreshold)
+         {
+         value = (trThreshold - mjitThreshold << 1) | 1;
```

Comment:
More parentheses required:
```
    value = ((trThreshold - mjitThreshold) << 1) | 1;
```

Reply:
Updated now, thank you for catching this.